ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml

```diff
@@ -209,6 +209,11 @@ tests:
   steps:
     cluster_profile: openstack-vh-mecha-az0
     workflow: openshift-e2e-openstack-perfscale-ingress-perf
+- as: e2e-openstack-network-perf
+  minimum_interval: 336h
+  steps:
+    cluster_profile: openstack-vh-mecha-az0
+    workflow: openshift-e2e-openstack-perfscale-network-perf
```

Comment:
I think you're missing the CNI (`ovn`) in the [job name](https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/)

Reply:
On another note: why `perfscale-network-perf` and not just `perfscale-network`?